pkgs/development/python-modules/testpath/default.nix

```diff
@@ -24,6 +24,12 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
+  # exe are only required when testpath is used on windows
+  # https://github.com/jupyter/testpath/blob/de8ca59539eb23b9781e55848b7d2646c8c61df9/testpath/commands.py#L128
+  preBuild = lib.optional (!stdenv.targetPlatform.isWindows) ''
```

Comment:
this should be lib.optionalString

Reply:
oops! Thank you! https://github.com/NixOS/nixpkgs/pull/224705